src/pages/VenuePage/VenuePage.tsx

```diff
@@ -89,10 +112,6 @@ export const VenuePage: React.FC = () => {
 
   const event = currentEvent?.[0];
 
-  const { isLoading } = useRelatedVenues({
-    currentVenueId: venueId,
-  });
-
```

Comment:
haven't checked other comments. Can you explain why is that changed? Don't we need the `isLoading` anymore here?

Reply:
This component doesn't use the related venues, so it makes no sense to show loading screen for children components. They can use `useRelatedVenues()` themselves the data and the flag and display their own loading message or icon.

Reply:
Accepted changes https://github.com/sparkletown/sparkle/pull/1713/commits/660c49d3e8c725ff2f62c860c8b9c86b326cf451